bin/render-build.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
```

Comment:
Please follow the layout and over user interface for `Categories` page. So the icon and details should be on left and expenses for the category are on right of each card.

Please follow the layout and over user interface for `transactions` page. So the icon and details should be on left and expenses are on right of each card. Moreover, total amount should be on top as it can be shown in the design.


Please follow the design(fonts, colors, and layout) for all the pages and app navigation.

Please render the list of transactions in ordered by the most recent for a given category on the transactions page.